tsl/src/compression/compress_utils.c

```diff
@@ -330,12 +334,13 @@ tsl_compress_chunk(PG_FUNCTION_ARGS)
 Datum
 tsl_decompress_chunk(PG_FUNCTION_ARGS)
 {
-	Oid uncompressed_chunk_id = PG_ARGISNULL(0) ? InvalidOid : PG_GETARG_OID(0);
+	Oid uncompressed_chunk_id = PG_GETARG_OID(0);
 	bool if_compressed = PG_ARGISNULL(1) ? false : PG_GETARG_BOOL(1);
 	Chunk *uncompressed_chunk = ts_chunk_get_by_relid(uncompressed_chunk_id, 0, true);
-	if (NULL == uncompressed_chunk)
-		elog(ERROR, "unknown chunk id %d", uncompressed_chunk_id);
 
+	/* SQL function `decompress_chunk` is declared as STRICT, so
```

Comment:
I would elide this comment. We have many `STRICT` functions and we don't generally have comments like this for other functions. The assertion conveys enough information IMO, and also protects against changes in `STRICT` that would make this comment outdated.

Reply:
Removed.